tools/generatechangelog/generatechangelog_test.go

```diff
@@ -0,0 +1,322 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+	"time"
+)
+
+const (
+	TAG_NAME = "v1.21.0"
+	TAG_DATE = "2024-03-20"
+)
+
+func TestGetLatestTag(t *testing.T) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Failed to get current working directory: %v", err)
+	}
```

Comment:
```suggestion

	require.NoError(t, err)
```

We use testify for all tests

Reply:
refactored to use `testify`